src/Okta.Sdk/Internal/DefaultRequestExecutor.cs

```diff
@@ -216,5 +254,18 @@ public Task<HttpResponse<string>> DeleteAsync(string href, IEnumerable<KeyValueP
 
             return SendAsync(request, cancellationToken);
         }
+
+        private HttpRequestMessage CreateMessage(HttpRequest httpRequest)
```

Comment:
Nit: I would rename this method to be more specific, something like `CreateHttpRequestMessage`.